<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlButtonGroup</class>
 <widget class="QGroupBox" name="ControlButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="sensorSetZero">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>传感器置零</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Setting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>自动控制</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="motorEnable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>电机使能</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="StartAndStop">
     <property name="text">
      <string>设备启动</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="sensorLink">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>传感器连接</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="motorDiable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>电机失能</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
